### PR TITLE
Feature caching dns server

### PR DESCRIPTION
**Pull request by jehov4**
Incorporates Issue #97 and Issue #98

### CODE REVIEW

File: 05_service_configuration/050_caching_dns_server/051_caching_dns_server.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Configure a Caching DNS Server
+This section is based on this [cloudflare article](https://www.cloudflare.com/learning/dns/what-is-recursive-dns/) and this [LinuxTeck guide](https://www.linuxteck.com/how-to-setup-caching-dns-server-in-centos-rhel-7-6/).
```
Comment (philippabele) [inline 05_service_configuration/050_caching_dns_server/051_caching_dns_server.md:2]: I think you dont need the word "this" to reference your sources. Just mention that this section is based on an cloudflare article and a how to guide from linuxteck as well as add the corresponding links. Sounds more academic :)

File: 05_service_configuration/050_caching_dns_server/052_dns_zones.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# DNS Zones
+Based on this this [Ubuntu Wiki](https://wiki.ubuntuusers.de/DNS-Server_Bind/#Das-Anlegen-der-Zonendateien) article, the examples are taken from there.
```
Comment (philippabele) [inline 05_service_configuration/050_caching_dns_server/052_dns_zones.md:2]: Please see my last comment about "this". It won't get better if you double the appreance of "this" :)

File: 05_service_configuration/050_caching_dns_server/052_dns_zones.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+10      IN      PTR     rechnername.domainname.
+200     IN      PTR     rechner1.domainname.
+201     IN      PTR     rechner2.domainname.
+```
+
+Here the `PTR` entry type comes to use. The numbers in the first row represent the last byte of the IP address.
+
+## Exercise:
+Read more about DNS on [Wikipedia](https://en.wikipedia.org/wiki/Domain_Name_System) if you want to learn more.
```
Comment (philippabele) [inline 05_service_configuration/050_caching_dns_server/052_dns_zones.md:94]: Anaphora of more.

**Review by philippabele — APPROVED** [review]

**Review by philippabele — CHANGES_REQUESTED** [review]
I unfortunately approved the PR with requested changes. I'm sorry for that mess. Please see my comments and fix them before PR again. Thanks.

File: 05_service_configuration/050_caching_dns_server/052_dns_zones.md (hunk 2 above)
Comment (jehov4) [reply 05_service_configuration/050_caching_dns_server/052_dns_zones.md]: Done

**Review by philippabele — APPROVED** [review]

**Pull request merged**